doctrine/event_listeners_subscribers.rst

```diff
@@ -150,8 +150,8 @@ interface and have an event method for each event it subscribes to::
         public function getSubscribedEvents()
         {
             return array(
-                'postPersist',
-                'postUpdate',
+                \Doctrine\ORM\Events::postPersist => 'postPersist',
+                \Doctrine\ORM\Events::postUpdate => 'postUpdate',
```

Comment:
I think we should import `Doctrine\ORM\Events` and use `Events::postPersist` and `Events::postUpdate` here instead.